hasher-matcher-actioner/hmalib/lambdas/actions/reactioner.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import json
+
+from hmalib.common.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+def lambda_handler(event, context):
+    """
+    This is the main entry point for reacting to ThreatExchange. The action evaluator
+    sends a reaction message by way of the reactions queue and here's where they're
+    popped off and dealt with.
+    """
+    for sqs_record in event["Records"]:
```

Comment:
I've seen else where us check for test events. Do we want to do that here as well?

Reply:
Sounds good! Done!